src/lib/permissions/interfaces.ts

```diff
@@ -1,6 +1,19 @@
 // --- Domain specific interfaces
-export * from './spaces/interfaces';
-export * from './forums/interfaces';
-export * from './proposals/interfaces';
-export * from './pages/interfaces';
-export * from './core/interfaces';
+import type { BaseForumPermissionsClient, PremiumForumPermissionsClient } from './forums/client/interfaces';
```

Comment:
i think we should get rid of or rename one of these: `permissions/interfaces.ts` or `permissions/core/interfaces.ts`. For instance, why can't PremiumPermissionsClient be defined in permissionsApiClient.class?